great_expectations_cloud/agent/agent.py

```diff
@@ -88,7 +88,7 @@ class GXAgent:
     def __init__(self: Self):
         agent_version: str = self.get_current_gx_agent_version()
         great_expectations_version: str = self._get_current_great_expectations_version()
-        print(f"GX Agent version: {agent_version}")
+        LOGGER.info(f"GX Agent version: {agent_version}")
```

Comment:
Why change this to a log?

Reply:
oh this should be reverted, I was just testing out logger

Reply:
reverted in 5b40a87